src/shared/Paths.js

```diff
@@ -1,3 +1,12 @@
-export const MOJANG_PATH = `${process.env.LOCALAPPDATA.replace(/\\/g, "/")}/Packages/Microsoft.MinecraftUWP_8wekyb3d8bbwe/LocalState/games/com.mojang/`;
```

Comment:
This specific change will probably require a file cache update so users on Windows and Linux can use the same cache.

Reply:
There's also some legacy code in different files which might not like the path separator not being a normal "/". (<path>.includes("something/"))

Reply:
> This specific change will probably require a file cache update so users on Windows and Linux can use the same cache.

It will be breaking change. I think the cache system shouldn't use absolute path at all.

> There's also some legacy code in different files which might not like the path separator not being a normal "/". (<path>.includes("something/"))

It can be fixed by modify legacy code..(just replace befor includes

Working in progress

Reply:
That looks good now 👍! I agree on the relatives paths, that‘s a lot cleaner. The lightning cache also needs an update but I think after that we should be good to push this to a pre-release for public testing